### ISSUE TITLE
A problem about graph tab and profile tab in tensorboard

### ISSUE THREAD

**Issue by tomtang110** [tensorflow/tensorboard#3388]
Consider Stack Overflow for getting support using TensorBoard—they have
a larger community with better searchability:

https://stackoverflow.com/questions/tagged/tensorboard

Do not use this template for for setup, installation, or configuration
issues. Instead, use the “installation problem” issue template:

https://github.com/tensorflow/tensorboard/issues/new?template=installation_problem.md

To report a problem with TensorBoard itself, please fill out the
remainder of this template.

## Environment information (required)

Chrome version 80.0.3987.132
tensorflow version 2.1.0
tensorboard version 2.1.0

**## Issue description**
I create a customized model, and when I use callback in model.fit function, I found the graph tap shows that failed normalizing names and the profile tap disappeared.
The code is shown below
`afm_model = AFM(**pnn_params)
model = afm_model.model` 
`check_path = './model_save/AFM/model.ckpt'
callback = [
            tf.keras.callbacks.TensorBoard(log_dir='./logs/AFM',histogram_freq=5000,
            write_graph=True, write_images=False,
            embeddings_freq=0, embeddings_layer_names=None,
            embeddings_metadata=None, embeddings_data=None, update_freq=500,profile_batch=10),
            tf.keras.callbacks.ModelCheckpoint(check_path,save_weights_only=True,verbose=1,save_freq=10000)
           ]`
`model.fit([Xi_train,Xv_train,y_train],y_train,
           validation_data = ([Xi_test,Xv_test,ids_test],ids_test),
            epochs=5,
            batch_size=pnn_params['batch_size'],
            callbacks = callback,
            shuffle=True)` 
![屏幕快照 2020-03-18 09 34 03](https://user-images.githubusercontent.com/33299522/76916586-d81bc500-68fb-11ea-98d5-60539494a9d0.png)
![屏幕快照 2020-03-18 09 35 59](https://user-images.githubusercontent.com/33299522/76916600-ebc72b80-68fb-11ea-8cac-43f2d2eaed0d.png)

**Comment by bileschi** [comment]
Hi @tomtang110 do you have a link to a logdir you can share or a complete package that we could execute to create one? It looks like the graph written by the algorithm was in a format that tensorboard did not expect.
